test/ProductConstructionService.DependencyFlow.Tests/PendingCodeFlowUpdatesTests.cs

```diff
@@ -46,7 +46,7 @@ public async Task PendingUpdatesUpdatablePrButNoNewBuild()
         {
             await WhenProcessPendingUpdatesAsyncIsCalled(build, isCodeFlow: true);
 
-            AndShouldHaveInProgressPullRequestState(build);
+            AndShouldHaveInProgressPullRequestState(build, nextCommitToProcess: null);
```

Comment:
Please make that parameter optional so we don't have to spell out the `nextCommitToProcess: null`